controllers/datamovementmanager_controller.go

```diff
@@ -315,12 +319,40 @@ func (r *DataMovementManagerReconciler) createOrUpdateServiceIfNecessary(ctx con
 	return nil
 }
 
+func (r *DataMovementManagerReconciler) updateLustreFileSystemsIfNecessary(ctx context.Context, manager *dmv1alpha1.DataMovementManager) error {
+	log := log.FromContext(ctx)
+
+	filesystems := &lusv1alpha1.LustreFileSystemList{}
+	if err := r.List(ctx, filesystems); err != nil && !meta.IsNoMatchError(err) {
+		return fmt.Errorf("list lustre file systems failed: %w", err)
+	}
+
+	for _, lustre := range filesystems.Items {
+		_, found := lustre.Spec.Namespaces[manager.Namespace]
+		if !found {
+			log.Info("Updating Lustre File System", "object", client.ObjectKeyFromObject(&lustre))
```

Comment:
It would be more descriptive if you say that you're adding the namespace here rather than just a generic update.